src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -147,6 +149,11 @@ export const CreateTransaction = ({
     initialFormData.isAccountFunded,
   );
 
+  const [isAccountUnsafe, setIsAccountUnsafe] = useState(
+    initialFormData.isAccountUnsafe,
+  );
+  const [isAccountMalicious, setIsAccountMalicious] = useState(false);
```

Comment:
use initialFormData to populate isAccountSafe to persist the data

We don't bother with `isAccountMalicious` as a user should be stuck on this screen if the address is malicious